src/components/commons/SkipToContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "@mui/material";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+const SkipToContent = () => {
+  const { t } = useTranslation();
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
+
+  const linkStyle: any = {
+    transition: "opacity 0.2s ease-out",
+    position: "relative",
+    opacity: isFocused ? "1" : "0",
+    zIndex: isFocused ? "1" : "-1",
+  };
+
+  return (
+    <div style={{ position: "relative" }}>
+      <Button
+        href="#main_content"
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+        style={linkStyle}
+      >
+        {t("mainPage.main.skipToContent")}
+      </Button>
```

Comment:
If you click this button on riepilogo-pagamento page, this wrong dialog appear:

![image](https://github.com/user-attachments/assets/297f8c61-d084-4257-9f47-871cd1cebf35)

To fix this problem you should replace href attribute with `onClick={() => document.getElementById("main_content")?.focus()}`, because some pages have a check to prevent user leaving payment that is triggered by href.
This is appened beacuse href adds a fragment to the url, which triggers popstate event used to show this dialog.

Reply:
i'm afraid we cannot do that, because the screen reader needs the href to be set on the skip-to-content link to be able to bypass (eventual) text content. Remember from an html point of view, the skip to content needs an <a> tag with href set (at least according to the documentation found online).

Reply:
I'm thinking of a possible work around

Reply:
done here e4eb02844ee93129becf1d979bc9a4e1dd6bfdee